intel_extension_for_transformers/llm/runtime/graph/models/opt/opt.cpp

```diff
@@ -97,7 +104,7 @@ static bool opt_model_eval_internal(model_context& lctx, const model_token* toke
   */
   struct ne_tensor* position = d_ne_new_tensor_1d(ctx0, NE_TYPE_I32, N);
   for (int i = 0; i < N; ++i) {
-    ((int32_t*)position->data)[i] = n_past + i + OPT_POS_EMBD_OFFS;
+    ((int32_t*)position->data)[i] = n_total + i + OPT_POS_EMBD_OFFS;
```

Comment:
@zhentaoyu Am I right here?

Reply:
logical pos is `n_past + i +offset`? `n_total` will be greater than `max_seq_length`?